packages/webview_flutter/webview_flutter_platform_interface/lib/src/method_channel/webview_method_channel.dart

```diff
@@ -266,6 +266,7 @@ class MethodChannelWebViewPlatform implements WebViewPlatformController {
       'userAgent': creationParams.userAgent,
       'autoMediaPlaybackPolicy': creationParams.autoMediaPlaybackPolicy.index,
       'usesHybridComposition': usesHybridComposition,
+      'backgroundColor': creationParams.backgroundColor?.value,
```

Comment:
This needs a corresponding test [here](https://github.com/flutter/plugins/blob/master/packages/webview_flutter/webview_flutter_platform_interface/test/src/method_channel/webview_method_channel_test.dart)

Reply:
I put the test in the `group('Tests on plugin.flutter.io/webview_<channel_id> channel', () {})` but maybe it would have been better to have them in a dedicated group for changes in creationParamsToMap()